Easydict/Swift/View/SettingView/Tabs/TabView/AdvancedTab.swift

```diff
@@ -172,6 +172,17 @@ struct AdvancedTab: View {
     @Default(.httpPort) private var httpPort
 
     @Default(.enableAppleOfflineTranslation) private var enableLocalAppleTranslation
+
+    /// Returns Color.green if `enableHTTPServer` is true,
+    /// returns Color.red otherwise.
+    private func getHttpIconColor() -> Color {
+        switch enableHTTPServer {
+        case true:
+            .green
+        case false:
+            .red
+        }
```

Comment:
```suggestion
        enableHTTPServer ? .green : .red
```

Reply:
The change has been made in https://github.com/tisfeng/Easydict/pull/696/commits/8a2cc5f712820acd6194adcc90d7d13e73e9fbf5, thank you for optimizing.